src/plugins/kibana/public/doc/controllers/doc.js

```diff
@@ -50,7 +50,7 @@ app.controller('doc', function ($scope, $route, es, timefilter) {
           values: [$route.current.params.id]
         }
       },
-      fields: computedFields.fields,
+      stored_fields: computedFields.fields,
```

Comment:
Since not all fields here would be stored presumably we need to split `computedFields.fields` into stored fields and _source fields and add them as two separate line items like this:

```
  storedFields: computedFields.storedFields,
  _source: {
    includes: computedFields.sourceFields
  }
```